staging/src/k8s.io/client-go/examples/README.md

```diff
@@ -0,0 +1,26 @@
+# client-go Examples
+
+This directory contains examples that cover various use cases and functionality
+for client-go.
+
+### Configuration
+
+- [**Authenticate in cluster**](./in-cluster-client-configuration): Configure a
+  client while running inside the Kubernetes cluster.
+- [**Authenticate out of cluster**](./out-of-cluster-client-configuration):
+  Configure a client to access a Kubernetes cluster from outside.
+
+### Basics
+
+- [**Managing resources with API**](./create-update-delete-deployment): Create,
+  get, update, delete a Deployment resource.
+
+### Advanced Concepts
+
+- [**Work queues**](./workqueue): Create a hotloop-free controller with the
+  rate-limited workqueue and the informer framework.
```

Comment:
Could you make "informer framework" a hyperlink to https://godoc.org/k8s.io/client-go/tools/cache#NewInformer?